athenacli/main.py

```diff
@@ -74,7 +74,16 @@ def __init__(self, region, aws_access_key_id, aws_secret_access_key,
         aws_config = AWSConfig(
             aws_access_key_id, aws_secret_access_key, region, s3_staging_dir, profile, _cfg
         )
-        self.connect(aws_config, database)
+
+        try:
+            self.connect(aws_config, database)
+        except:
```

Comment:
How about catch the `Exception` and put it into the error message?

e.g. https://github.com/dbcli/athenacli/blob/master/athenacli/main.py#L329-L332

And also log the stack in the log file.

Reply:
Added trace and error in the log. 

I didn't intentionally print the error in the message, but rather gave a pointer to the user to check the logs if they are interested.